docs/user_guide/ug_general_concepts.md

```diff
@@ -2,40 +2,97 @@
+The granularity is important : the line is composed by several section. For each section, there are several cables. Each cable can be divided into spans, which is the lowest scale of calculus.
 
-[scheme granularity]
-Câble : Une phase peut être composée d’un unique câble ou d’un faisceau de câbles. Dans la suite on parle de câble de manière générique : phase constituée d'un câble simple, phase constituée d'un faisceau de câble, ou câble de garde.
-
-The algorithms are switching between the different levels during the resolution
+The algorithms are switching between the different levels during the resolution.
 
 ## Definition
 
 ![Image not available](./assets/powerline_definitions.drawio.png "Powerlines definitions")
 
-As you can see below, the study cable is hanging to different suspension strings and start/stopping to a dead end string.
+As described below, the study cable is hanging to different suspension strings and start/stopping to a tension support.
 
 Occasionnaly, some line angles can be found into a section. In this case, we are considering the axe of the pylone (i.e. the arm direction) as the angle bisector.
 
 ## Frames
 
 Depending on the physical models used in the algorithms, the 3D objects, coordinates, forces and moments are projected onto different planes.
 
-The comprehension of those planes are not mandatory to use the package, but needed if you want to dive into the different physical models used.
+Understanding these planes is not required to use the package, but it is necessary if you wish to explore the various physical models employed.
+
+However, the different frames defined below are necessary to display the results. Therefore, we are outlining the transformation operations to transition between them.
 
-However the different frames defined below are needed to display the result : that is why we are defining the transformation operations to go from one to another.
+### Earth frame
 
-### Span frame
+#### Base frame
+
+The Earth frame  $\mathcal{R}_{earth}$ is defined as the GPS coordinates system :
+
+- x along the west-east axis, facing east
+- y along the north-south axis, facing north
+- x and y in the plan, z is orthogonal to this plan
+- the origin of the frame is the crossing of equator and prime meridian.
+
+!!! important
+
+    For now, only the local frame is considered (in blue in the figure), with the hypotheses that the origin is a plane translation relative to the O point in the figure.
+
+![Image not available](./assets/earth_frame.drawio.png "Earth frame")
+
+
+#### Georeferencing
+
+Other coordinates system may be added in the future.
+
+### Support frame
+
+There are two frames associated with the support:
+
+- The tower body frame: $\mathcal{R}_{towerbody}$
+- The crossarm frame: $\mathcal{R}_{crossarm}$
+
+Those frames are oriented with z axis up and x axis along the crossarm, pointing outwards from the pylon.
```

Comment:
Those frames are oriented with the z-axis up and the x-axis along the crossarm, pointing outward from the pylon.

En dehors de cette petite correction, je me demandais, c'est pas x dans le sens de la span ?

Reply:
As you mentionned we can set x along the span, so y along the arm and pointing inward from the pylone